gtfs_aggregator_checker/__init__.py

```diff
@@ -26,12 +26,13 @@ def clean_url(url):
     return urllib.parse.urlunparse(url)
 
 
-def main(
-    yml_file=typer.Argument("agencies.yml", help="A yml file containing urls"),
-    csv_file=typer.Option(None, help="A csv file (one url per line)"),
-    url=typer.Option(None, help="URL to check instead of a file",),
-    output=typer.Option(None, help="Path to a file to save output to."),
-    verbose: bool = typer.Option(False, help="Print a result table to stdout"),
+def check_feeds(
+    yml_file=None,
+    csv_file=None,
+    url=None,
+    output=None,
+    verbose=False,
+    clear_cache=False,
```

Comment:
This clear_cache argument doesn't seem to do anything, or am I missing something? Seems like since the README says to delete the file manually, maybe a CLI flag isn't needed.